mmcv/ops/masked_conv.py

```diff
@@ -61,7 +61,13 @@ def forward(ctx, features, mask, weight, bias, padding=0, stride=1):
                 kernel_w=kernel_w,
                 pad_h=pad_h,
                 pad_w=pad_w)
-
+            # We change the order of parameters because:
+            # This overload of addmm is deprecated:
```

Comment:
Which PyTorch version deprecated the signature?

Reply:
1.5.0